docs/organise/online/index.md

```diff
@@ -62,9 +62,15 @@ copy that can be uploaded to youtube.
 
 ### Streaming
 
-Worth noting twitch does not transcode unless you are a partner, so your input
-stream is sent directly to hackers. Streaming in 1080p 60fps might exclude
-hackers with weaker internet connections.
+Worth noting Twitch does not transcode unless you are a partner, so your input
+stream is sent directly to hackers.
+Streaming in 1080p 60fps without transcoding might exclude hackers with weaker
+internet connections.
+Working with Hackathons UK or other similar organisations can be a solution to
+this as they have the potential to offer partnered channels to stream on.
```

Comment:
I would update this to clarify that Hackahtons UK only has one twitch channel.

```suggestion
Working with Hackathons UK or other similar organisations can be a solution to
this as they have the potential to offer a partnered channel to stream on.
```

Reply:
This was due to the or as I'm not sure the situation with other orgs, but happy to accept if there is no info on others yet.